frontend/app/src/locales/es.json

```diff
@@ -2222,6 +2222,31 @@
         "empty": "The decimal separator cannot be empty"
       }
     },
+    "external_service_setting": {
+      "label": {
+        "query_retry_limit": "查询重试限制",
```

Comment:
¡Un momento! ¡Esto no parece español! (used deepl for this)
Wait a minute!!! this does not look like Spanish! 🇪🇸 🤣 

This we will have to revert

Reply:
my bad, should just let chatgpt handle it 😅